hat/assets/js/apps/Iaso/domains/instances/components/InstancesFilesListComponent.tsx

```diff
@@ -172,6 +196,14 @@ const InstancesFilesList: FunctionComponent<Props> = ({
                     setCurrentIndex={newIndex =>
                         handleSetCurrentIndex(newIndex, 'images')
                     }
+                    link={
+                        showLink ? (
+                            <FormSubmissionLink
+                                images={sortedFiles.images}
+                                imageIndex={currentImageIndex}
+                            />
+                        ) : null
+                    }
```

Comment:
Overall, I think it would be cleaner to have an `ImageGalleryLink` component that just receives the link url.
That way, you can just pass the url here without changing the logic related to `showLink`.